examples/guided.py

```diff
@@ -342,18 +455,34 @@ def perform_installation(mountpoint):
 				installation.enable_service('systemd-resolved')
 
 			if archinstall.arguments.get('audio', None) != None:
-				installation.log(f"This audio server will be used: {archinstall.arguments.get('audio', None)}", level=logging.INFO)
+				installation.log(
+					f"This audio server will be used: {archinstall.arguments.get('audio', None)}",
+					level=logging.INFO,
+				)
 				if archinstall.arguments.get('audio', None) == 'pipewire':
 					print('Installing pipewire ...')
 
-					installation.add_additional_packages(["pipewire", "pipewire-alsa", "pipewire-jack", "pipewire-media-session", "pipewire-pulse", "gst-plugin-pipewire", "libpulse"])
+					installation.add_additional_packages(
+						[
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability. I don't care much for empty lines like this. It serves no use unless you're a PHP/C/C++ developer coming in to Python. Again, this is very opinionated and what I think doesn't mean I'm right. But this feels useless to me and a complete waste of code lines.